### PR TITLE
Make Basic HTTP authentication configurable

### PR DESCRIPTION
**Pull request by csomh**
Add configuration keys for user and password.

Use Basic authentication upon creating results,
when the credentials are configured.

Signed-off-by: Hunor Csomortáni <csomh@redhat.com>

### CODE REVIEW

**Comment by mikebonnet** [comment]
+1 Looks good!

**Comment by mprahl** [comment]
@csomh looks elegant. Thanks for the PR.

File: openshift/resultsdb-updater-test-template.yaml [hunk 1]
```diff
@@ -64,7 +64,10 @@ objects:
         'stomp_ssl_key': '/etc/resultsdb/umb-consumer.key.pem',
         'resultsdb-updater.log_level': logging.DEBUG,
         'resultsdb-updater.resultsdb_api_url': '${RESULTSDB_API_URL}',
-        'resultsdb-updater.resultsdb_api_ca': None,
+        # TODO make certificate verification work
```
Comment (mprahl) [inline openshift/resultsdb-updater-test-template.yaml]: As long as you don't use the Python requests from pip you won't need this set as the system CA bundle should work just fine unless the CA is not installed of course.

Comment (csomh) [reply openshift/resultsdb-updater-test-template.yaml]: The CA is not installed.

**Pull request merged**